### ISSUE TITLE
Can the model be trained successfully if log probabilities (logprobs) or probabilities (probs) are used as inputs instead?

### ISSUE THREAD

**Issue by cdxzyc** [jxmorris12/vec2text#62]
In the paper, logits are used as inputs. Can the model be trained successfully if log probabilities (logprobs) or probabilities (probs) are used as inputs instead?

**Comment by jxmorris12** [comment]
Yes! We show that logprobs are actually slightly more useful than logits according to our experiments in Table 9 (it's buried in the appendix). 
<img width="769" alt="Screenshot 2024-08-14 at 11 19 34 AM" src="https://github.com/user-attachments/assets/296b7b94-3d4d-427e-9913-32b5f223ee3d">

It doesn't look like there is an easy option to flip the softmax off in the code as far as I can tell, but you can just edit these lines: https://github.com/jxmorris12/vec2text/blob/f1369cbfe6bf9e216ac2a3758369c165278c9046/vec2text/models/inversion_from_logits.py#L177-L181
